tests/meta/workflow/test_balsamic.py

```diff
@@ -0,0 +1,40 @@
+"""Tests for BALSAMIC analysis"""
+from cg.exc import BalsamicStartError
+
+from cg.meta.workflow.balsamic import BalsamicAnalysisAPI
+
+
+def test_get_verified_gender():
+    """Tests gender extraction from a sample dictionary"""
+
+    # GIVEN a sample object
+    sample_obj = {
+        "ACC0000A0": {"gender": "female"},
+        "ACC0000A1": {"gender": "female"},
+    }
+
+    # WHEN extracting the gender
+    retrieved_gender = BalsamicAnalysisAPI.get_verified_gender(sample_obj)
+
+    # THEN gender must match the expected one
+    assert retrieved_gender == "female"
+
+
+def test_get_verified_gender_error(caplog):
+    """Tests gender extraction from a sample dictionary when two different gender has been provided"""
+
+    # GIVEN a sample object with different genders
+    sample_obj = {
+        "ACC0000A0": {"gender": "male"},
+        "ACC0000A1": {"gender": "female"},
+    }
+
+    # WHEN extracting the gender
+    try:
+        BalsamicAnalysisAPI.get_verified_gender(sample_obj)
+    except BalsamicStartError:
```

Comment:
Can also be written as
```suggestion
    with pytest.raises(BalsamicStartError):
        BalsamicAnalysisAPI.get_verified_gender(sample_obj)
```

Reply:
Changed: [3e60993](https://github.com/Clinical-Genomics/cg/pull/1493/commits/3e6099310b9aad91bfc2303781295e015b8b389b)